istio-control/istio-discovery/values.yaml

```diff
@@ -96,8 +96,14 @@ pilot:
     ingressClass: istio
 
   telemetry:
-    # Will not define mixerCheckServer and mixerReportServer
-    enabled: true
+    mixerv1:
```

Comment:
kill the name Mixer. We should not add anything that says `mixer` anywhere :-)

1. `telemetry.enabled: false` should mean turn off all telemetry.
so please restore that option.

2. rename options to `v1` and `v2` 
Fix comment.
v1.enabled `will` define mixerCheck and report servers.

Reply:
what if one specifies telemetry.enabled: true and v1.enabled: true and v2.enabled: true? Which should actually take effect? We can add corresponding validation logic in operator but not sure how to deal with the values.yaml case.